codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/customizations/PresignableUrlIntegrationTests.kt

```diff
@@ -13,43 +13,6 @@ import software.amazon.smithy.swift.codegen.core.GenerationContext
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
 
```

Comment:
The cases tested here that have been removed are now tested under `SigV4AuthSchemeTests`.